packages/compass-auto-updates/.depcheckrc

```diff
@@ -0,0 +1,14 @@
+ignores: [
+  "@babel/cli",
+  "@babel/register",
+  "core-js",
+  "electron-packager",
```

Comment:
I'm like 85% sure that plugins don't need any dependencies on `electron-packager` (or `electron-builder` that can be seen in some other ones), maybe we can try cleaning them up in one of them and checking if everything still works there?

Reply:
Sure, If the build works in the CI here that would be already a good sign right?

Reply:
Yeah, it would be enough to check that it's okay to remove them (I forgot for a sec that we have CI in PR branches now, lol)